src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { getParameters } from 'codesandbox/lib/api/define';
+
+const html = `
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="theme-color" content="#000000">
+	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+	<title>React App</title>
+</head>
+<body>
+	<noscript>
+		You need to enable JavaScript to run this app.
+	</noscript>
```

Comment:
Is this needed? Gotta think the user has JS if they got this far.

Reply:
This is a lift from the `create-react-app` templates. If someone were to publish their sandbox and a consumer didn't have JS this would be necessary. Since it's CRA best practice should we leave it?

Reply:
Ah, good point.